modules/Bio/EnsEMBL/ProteinFeature.pm

```diff
@@ -56,10 +56,11 @@ package Bio::EnsEMBL::ProteinFeature;
 use strict;
 
 use Bio::EnsEMBL::FeaturePair;
+use Bio::EnsEMBL::BaseAlignFeature;
 use Bio::EnsEMBL::Utils::Argument qw(rearrange);
 
 use vars qw(@ISA);
-@ISA = qw(Bio::EnsEMBL::FeaturePair);
+@ISA = qw(Bio::EnsEMBL::BaseAlignFeature);
```

Comment:
again, use parent is better.
`use parent qw(Bio::EnsEMBL::BaseAlignFeature);`

instead of
`use vars qw(@ISA);`
`@ISA = qw(Bio::EnsEMBL::BaseAlignFeature);`